tests/unit/Endpoints/Webhooks/ProductHandlerTest.php

```diff
@@ -68,58 +47,121 @@ public function testCreateProduct()
         $this->assertTaskNotCount(0, 'There are suppose to be any tasks');
     }
 
-    public function testUpdateProduct()
+    public function testUpdateProductByDateUpdated()
     {
-        // Handle the product creation hook event so there is a product to update
-        $creation_options = $this->handle_hook_request(
-            self::CREATE_DATADUMP_DIRECTORY,
-            self::CREATE_DATADUMP_HOOK,
-            'ShopModule::ShopProduct',
-            'events'
-        );
-        // Fetch the product that was created by the creation hook
-        $wc_products = wc_get_products(
-            [
-                'post_type' => 'product',
-                'meta_key' => 'storekeeper_id',
-                'meta_value' => $creation_options['id'],
-            ]
-        );
-        $this->assertCount(
-            1,
-            $wc_products,
-            'Actual size of the retrieved product collection is not equal to one'
-        );
+        $createdProduct = $this->mockCreateProductRequestWithTest();
```

Comment:
i do not understand this test, all of those should be different events, yet you use same dump for all subtests

Reply:
Added new hook event dumps for `product stock, prices, upsell and cross sell`.